src/tidysic.py

```diff
@@ -1,24 +1,28 @@
 #!/usr/local/bin/python3
 
 from tinytag import TinyTag
+import eyed3
 import os
 from __argparse__ import create_parser
-from __os_utils__ import (file_extension,
+from __os_utils__ import (file_extension, filename,
                           create_dir, get_audio_files, move_file, lint_folders)
 
 
-def guess_artist(title):
+def guess_file_metadata(filename):
     '''
+    Guess the artist and title based on the filename
     '''
     try:
-        artist_separator = title.find('-')
-        if artist_separator > 0:
-            artist = title[0:artist_separator].lstrip()
-            new_title = title[artist_separator + 1:-1].lstrip()
-
-            return [artist, new_title]
+        # Artist and title are often seprated by ' - '
+        separator = filename.find(' - ')
+        if separator > 0:
+            artist = filename[0:separator].lstrip()
+            title = filename[separator + 2:len(filename)].lstrip()
+
+            return (artist, title)
+        return (None, None)
```

Comment:
I would prefer an `else` here if you don't mind.

Reply:
I don't (:

Reply:
OK no worries